docs/getting-setup.md

```diff
@@ -52,8 +52,8 @@ cd debugger.html
 yarn start
 ```
 
-*What does this do?*  
-This command starts a [development server](../packages/devtools-local-toolbox/README.md).
+*What does this do?*
+This command starts a [development server](https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-local-toolbox/README.md).
```

Comment:
@juliandescottes you should know this by now!

We move quickly here, so the link is already out of date :)

should be https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-launchpad/README.md